README.md

```diff
@@ -1,13 +1,82 @@
-# Metadev-Client-Application-Example
+Metadev-Client-Application-Example
+===========
 
-## Installing Docker 
+## Introduction.
 
-Follow [docker docs](https://docs.docker.com/get-docker/) to install docker.
+Metadev-client-example is an example project which uses the Metadev client framework. The goal of the framework is to reduce the code written for frontend development. The framework is best suited for [ERP](https://en.wikipedia.org/wiki/Enterprise_resource_planning) apps which have a ton of forms.
+
+
+**Metadev** is an abbreviation for Metadata driven development.
+
+---
+
+## How it works?
+
+The project is built using Typescript and Angular. Also the framework uses [Material Design Components](https://material.angular.io/). 
+
+[Metadev-Server-Example](https://github.com/supreethavadhani/metadev-server-example) generates the models using json. The generated files are then copied into to the project.
```

Comment:
Generates the models using JSON for what purpose exactly? And then are the models the same as the files or different? Not sure I'm missing a step here...

Reply:
Yes models are same as the file. Thats a good catch. I will make the changes 👍